.github/workflows/test.yml

```diff
@@ -15,11 +15,19 @@ jobs:
       with:
         python-version: 3.8
 
-    - name: Install Tox
-      run: pip install tox
+    - name: Install Dependencies
+      run: pip install .[lint]
 
-    - name: Run Tox
-      run: TOXENV=lint tox -r
+    - name: Run Black
+      run: black --check --force-exclude=vyper/version.py ./vyper ./tests ./setup.py
+
+    - name: Run flake8
+      run: flake8 ./vyper ./tests
```

Comment:
why not also `setup.py`?

Reply:
I was replicating the coverage from `tox.ini` so I am not sure why `setup.py` was originally excluded from `flake8`.

https://github.com/vyperlang/vyper/blob/dd3dcb86ddefe2929bf8f888ededef18ac59f2d4/tox.ini#L46-L52

I did some digging, and noticed this [commit](https://github.com/vyperlang/vyper/commit/9ec8f9de6c7d2aced7b57a0c4cf2528aa6b44c17#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449) where `setup.py` was added to `black` and `isort` but specifically left out for `flake8`. However, it is not explained why `flake8` did not include `setup.py`.

In any event, I ran `flake8` on `setup.py` locally, and only one issue is flagged. 
```
setup.py:56:20: W605 invalid escape sequence '\.'
setup.py:56:25: W605 invalid escape sequence '\d'
```
https://github.com/vyperlang/vyper/blob/dd3dcb86ddefe2929bf8f888ededef18ac59f2d4/setup.py#L56
It seems minor and can be resolved by the following change
```
return re.sub(r"\.dev\d+", "", version_str) 
```

Given the above, should I proceed to add `setup.py` to `flake8`?

Reply:
Yes - makes sense to be the same as the others

Reply:
Updated!